### PR TITLE
Funfact

### PR DESCRIPTION
**Pull request by aannie-y**
Change 'greeting' to 'fun fact' functionality. Also fixed up some parts of index.html.

### CODE REVIEW

**Review by lilian-hunt — COMMENTED** [review]
lgtm

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -85,33 +104,29 @@
   font-size: 20px;
 }
 
-.gallery:hover {
+div.gallery:hover {
```
Comment (KathleenQ) [inline portfolio/src/main/webapp/style.css]: Why do you add "div" back? I'm not sure whether you need it. I deleted them for my codes and everything works fine.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -85,33 +104,29 @@
...
   transform: scale(1.1);
   opacity: 0.7;
 }
 
-.gallery img {
-  width: 100%;
+div.gallery img {
+  width:100%;
```
Comment (KathleenQ) [inline portfolio/src/main/webapp/style.css]: A space before "100%"?

**Review by KathleenQ — COMMENTED** [review]
Looks good to me generally. I only have two small confusion points.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -14,22 +15,31 @@
...
 .cover {
   background-image: url("images/wallpaper.jpg");
   background-color: #90CAF9;
   background-position: center bottom;
   background-repeat: no-repeat;
   background-attachment: fixed;
   background-size: cover;
-  height: 400px;
+  height: 500px;
+  width: 100%;
```
Comment (jupi3200) [inline portfolio/src/main/webapp/style.css]: Is width: 100% useful? If you drop this line will there be any difference?

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -74,31 +102,49 @@
...
 }
+
 .gallery:hover {
   transform: scale(1.1);
   opacity: 0.7;
 }
+
 .gallery img {
-  width:100%;
+  width: 100%;
```
Comment (jupi3200) [inline portfolio/src/main/webapp/style.css:112]: Does the 'width: 100%' have any impact? If you remove this line will the result be the same?

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -1,19 +1,17 @@
...
-      <a href="#home" class="active">Home</a>
-      <a href="#about">About me</a>
-      <a href="#project">Projects</a>
-      <a href="#hobbies">What I enjoy</a>
-      <a href="#gallery">Gallery</a> 
-      <a href="#blog">Blog Posts</a>
+        <a href="#home" class="active">Home</a>
+        <a href="#about">About me</a>
+        <a href="#project">Projects</a>
+        <a href="#galleryDiv">Gallery</a>
```
Comment (tripd22) [inline portfolio/src/main/webapp/index.html]: Nit: [Indentation](https://google.github.io/styleguide/htmlcssguide.html#Indentation)

File: portfolio/src/main/webapp/script.js [hunk 6]
```diff
@@ -15,34 +15,36 @@
...
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts = [
+    'My childhood dream job was to be a police officer!',
+    'I am a huge fan of the Marvel universe!',
+    'My favourite superhero is Iron Man!', 'I do Taekwondo',
+    'I love playing tennis', 'I recently started to learn tricking!'
+  ];
 
   // Pick a random greeting.
```
Comment (tripd22) [inline portfolio/src/main/webapp/script.js]: Nit: Update comment.

File: portfolio/src/main/webapp/script.js [hunk 7]
```diff
@@ -13,36 +13,38 @@
...
-    navbar.classList.add('sticky');
-  } else {
-    navbar.classList.remove('sticky');
+  // Add sticky class to navbar when you reach its scroll position.
+  // Remove "sticky" when you leave scroll position.
+  function stickyNav() {
+    if (window.pageYOffset >= sticky) {
+        navbar.classList.add('sticky');
+    } else {
+        navbar.classList.remove('sticky');
```
Comment (tripd22) [inline portfolio/src/main/webapp/script.js]: Nit: indentation of navbar.classList... is 4 spaces.

**Review by jupi3200 — APPROVED** [review]

**Pull request merged**